citadel/indico_citadel/templates/event_category_warning.html

```diff
@@ -0,0 +1,10 @@
+{% from 'message_box.html' import message_box %}
+
+{% call message_box('warning', large_icon=true, fixed_width=true) %}
+    {% trans -%}
+        <b>This category contains a large number of events.</b>
+        <br/>
```

Comment:
We don't use XHTML -> `<br>`

Reply:
I would split the two parts into separate trans blocks. What way all the HTML can stay outside which is generally nicer :)